test/check-application

```diff
@@ -105,6 +105,12 @@ class TestFiles(testlib.MachineCase):
         b.wait_visible("[data-item='not-hidden']")
         b.wait_not_present("[data-item='.hiddenfile']")
 
+        # Symlink to `.` and `..` work and get shown as directories
+        m.execute("ln -sf . /home/admin/dot")
+        b.wait_visible("[data-item='dot'].symlink.folder")
+        m.execute("ln -sf .. /home/admin/dotdot")
+        b.wait_visible("[data-item='dotdot'].symlink.folder")
```

Comment:
Adding a test that we can `b.mouse("[data-item='dotdot'], 'dblick')` and then end up in /home would be nice if its not too much work.

Reply:
I actually expect us to end up in `/home/admin/dotdot` (which is the same as `/home` of course).  This is a UI question.